bin/run-docker.sh

```diff
@@ -82,7 +82,7 @@ case "$subcommand" in
                 WP_VERSION=${WP_VERSION} PHP_VERSION=${PHP_VERSION} docker compose up app
                     ;;
                 t )
-                docker-compose run --rm \
+                docker compose run --rm \
```

Comment:
Docker changed how compose is executed. I'm not sure if I was the only one having the issue, but if y'all aren't having issues this probably doesn't need to carry over.